updater/lib/dependabot/updater/operations/refresh_security_update_pull_request.rb

```diff
@@ -157,6 +159,20 @@ def check_and_update_pull_request(dependencies)
             requirements_to_unlock: requirements_to_unlock
           )
 
+          # Dependabot::Experiments.register(:existing_pr_version_match, true)
+
+          if Dependabot::Experiments.enabled?(:existing_pr_version_match) && (lead_dep_name &&
+            job.existing_pull_requests && pr_lead_dep_latest_ver(lead_dep_name,
+                                                                 lead_dep_latest_available_ver.to_s))
+
+            msg = "Lead dependency (#{lead_dep_name}) version (#{lead_dep_latest_available_ver}) is " \
```

Comment:
Do you need to assign the message to variable. You can directly log it since you are not using it for other purposes?

Reply:
code block removed